### PR TITLE
Update platform to 5.15-22.08

### LINKED ISSUE #80: This uses org.kde.Platform//5.15 which is end of life
Please update the runtime from org.kde.Platform//5.15 to a current one.

### PR DESCRIPTION
**Pull request by farribeiro**
~~And change `app-id` to `id`~~

CC @joao-p-marques

### CODE REVIEW

File: io.jamulus.Jamulus.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-app-id: io.jamulus.Jamulus
+id: io.jamulus.Jamulus
```
Comment (joao-p-marques) [inline io.jamulus.Jamulus.yaml:1]: Could you reference the docs to explain why this change is necessary?

**Review by joao-p-marques — COMMENTED** [review]
Thank you very much! Just a small doubt.

File: io.jamulus.Jamulus.yaml (hunk 1 above)
Comment (farribeiro) [reply io.jamulus.Jamulus.yaml:1]: See #81

Comment (farribeiro) [reply io.jamulus.Jamulus.yaml]: Some flatpak Devs says app prefix was deprecated. But no one send feedback about this in matrix Room

**Comment by farribeiro** [comment]
Free to free to accept or reject the PR

File: io.jamulus.Jamulus.yaml (hunk 1 above)
Comment (farribeiro) [reply io.jamulus.Jamulus.yaml]: Ping @gasevin @TingPing

Comment (joao-p-marques) [reply io.jamulus.Jamulus.yaml]: Seems good enough 👍  Have you tested if everything works with this new build?

Comment (farribeiro) [reply io.jamulus.Jamulus.yaml]: Works

**Review by joao-p-marques — CHANGES_REQUESTED** [review]
Can you rebase please?

**Review by joao-p-marques — APPROVED** [review]
